### ISSUE TITLE
Git for Windows 2.16.3 gives fatal: UriFormatException encountered, 2.16.2 works fine

### ISSUE THREAD

**Issue by hvdijk** [git-for-windows/git#1599]
- [X] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.16.3.windows.1
cpu: x86_64
built from commit: 5d726e05e4c0ff9e11374e64e21d8300b0291c6d
sizeof-long: 4
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.16299.309]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: VIM
Path Option: Cmd
SSH Option: OpenSSH
CURL Option: OpenSSL
CRLF Option: CRLFCommitAsIs
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Enabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

None.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Same behaviour from bash, cmd, and when invoked by GitExtensions's GUI.

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
git remote update
```
 - What did you expect to occur after running these commands?

```
Fetching origin
```

 - What actually happened instead?

```
Fetching origin
fatal: UriFormatException encountered.
   queryUrl
```

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

The URL is private, but I can say it is of the form `https://tfs.company.tld/tfs/Company/Project/_git/Project%20Name` where I suspect the `%20` is causing the problem. I attempted to set up a public repository to demonstrate the problem, but Github doesn't accept such characters in repository names.

**Comment by hvdijk** [comment]
This is caused by Git Credential Manager v1.15.0. Installing Git for Windows 2.16.3, and then installing Git Credential Manager v1.14.0 over it, works. So consider this report about a broken version being bundled. I will additionally report it there.

**Comment by hvdijk** [comment]
This turned out to not be about the configured URL itself, but a modified URL which includes username and password information. The `%20` was a red herring. It was the fact that the configured username was in `DOMAIN\username` format. This is supposed to have already been fixed by Microsoft/Git-Credential-Manager-for-Windows#601 so should make it into the next release.

**Comment by whoisj** [comment]
This issue has been fixed and the patched binary released as [v1.15.1](https://github.com/Microsoft/Git-Credential-Manager-for-Windows/releases/tag/v1.15.1).

**Comment by hvdijk** [comment]
No, what 1.15.1 (released five days ago) fixes is very closely related, it's another character in usernames that didn't work, but the issue I had (fixed three days ago) hasn't made it into a release yet.

**Comment by dscho** [comment]
@hvdijk no promises, but: we're working hard on getting this fixed in time for Git for Windows v2.17.0 (which is due some time next week). 🤞

**Comment by brentforder-qhr** [comment]
It's fixed in Git Credential Manager for Windows 1.15.2
https://github.com/Microsoft/Git-Credential-Manager-for-Windows/releases/tag/v1.15.2

**Comment by xtxtop** [comment]
my issue is the same to you.....
